src/components/ShowOTW.test.js

```diff
@@ -0,0 +1,22 @@
+import { render, screen } from "@testing-library/react";
+import ShowOTW from "./ShowOTW.js"; 
+
+const sampleShow = {
```

Comment:
At some point we could probably move the sample show into `globals.js`, or even create a new file called `test-utils.js` or something. Definitely not crucial right now though.